packages/core/src/App/Components/Elements/VerticalTabs/vertical-tab.jsx

```diff
@@ -12,9 +12,13 @@ class VerticalTab extends React.Component {
     }
 
     setSelectedIndex = ({ list, selected_index, is_routed, current_path }) => {
-        const index = typeof selected_index === 'undefined'
-            ? (is_routed ? list.indexOf(list.find(item => (item.path === (current_path || item.default)))) || 0 : 0)
-            : selected_index;
+        let index;
+        if (typeof selected_index === 'undefined') {
+            index = is_routed ?
+                list.indexOf(list.find(item => (item.path === (current_path || item.default)))) || 0 : 0;
+        } else {
+            index = selected_index;
+        }
```

Comment:
Did you just change this from ternary to if-else statement, or there is a change involved which I can't see?

Reply:
Lint error, my PR wasn't build because of this.

Reply:
Yeah just ternary to if-else